nss-day-cohort-22/Class-Website
# Add professional photos for all classmates into images/classmates,…

… added group photos into images/CohortGroupPhotos

## Link to Ticket

no direct ticket

## Description of Proposed Changes

- Added all the professional photos for classmates and group photos
- Added the links to images for my (Greg Lawrence) listing in the cohort.json
- Added group photos to main page. (We need to replace them with higher resolution versions)


## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-AddingPhotos
```
1. Load http-server 
1. Check that you see a group photo, and click through the slideshow to view the other two fun group photos.
1. check that my photo is showing and click on my photo and make sure the fun photo is displaying


## Impacted Areas in Application

List general components of the application that this PR will affect:

* images/classmates/xxx.jpg
* images/CohortGroupPhotos
* cohort.json
* index.html

## Mentions @username

Tag users that need to review this code
@krnorris65 @Gward2489 @john-dulaney
